feg/gateway/services/aaa/session_manager/client_api.go

```diff
@@ -17,11 +17,12 @@ package session_manager
 import (
 	"errors"
 	"fmt"
-	"github.com/golang/glog"
-	"golang.org/x/net/context"
 
 	"magma/feg/gateway/registry"
 	"magma/lte/cloud/go/protos"
+
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
```

Comment:
it should be:

```
  import (
	"errors"
	"fmt"

	"github.com/golang/glog"
	"golang.org/x/net/context"

	"magma/feg/gateway/registry"
	"magma/lte/cloud/go/protos"
  )
```

here, just add an empty line after "fmt"

Reply:
don't we add first magma dependencies and then external dependencies?

Reply:
for FeG module, the order was: std pkgs -> third party packages -> magma packages. it's what I see in most of the sources there